docs/04_protocol/02_transactions_protocol.md

```diff
@@ -363,15 +363,18 @@ The `status` field is an 8-bit enumeration type that can hold one of the followi
 * `executed` - transaction succeeded, no error handler executed.
 * `soft_fail` - transaction failed, error handler succeeded.
 * `hard_fail` - transaction failed, error handler failed.
-* `delayed` - transaction scheduled for future execution.
+* `delayed` - transaction delayed by user for future execution.
 * `expired` - transaction expired, CPU/NET refunded to user.
 
+[[info | "delayed" status]]
```

Comment:
it is ok to use ticks instead of double quotes...? I did not tried ... does it have a nicer effect than the double quotes?

Reply:
The backticks make the word appear white on the callout's blue background, which doesn't look pleasing especially on a title. Bold or italics didn't work so double quotes did the trick.